internal/mode/static/nginx/config/generator.go

```diff
@@ -47,11 +51,10 @@ const (
 
 	// mainIncludesConfigFile is the path to the file containing NGINX configuration in the main context.
 	mainIncludesConfigFile = mainIncludesFolder + "/main.conf"
-)
 
-// ConfigFolders is a list of folders where NGINX configuration files are stored.
-// Volumes here also need to be added to our crossplane ephemeral test container.
-var ConfigFolders = []string{httpFolder, secretsFolder, includesFolder, mainIncludesFolder, streamFolder}
```

Comment:
Do we not need to clear the config folders on startup anymore? Did that change with the init container?

Reply:
The only case where we would need to clear folder is if the control plane container only restarts, but nginx does not.

I'm not sure how common of an edge case this is, and this issue will go away in our 2.0 release anyway. But yes, this was causing issues where the files copied by the init container were being deleted.

Reply:
I see it was added here: https://github.com/nginxinc/nginx-gateway-fabric/pull/807 but I don't see mention of a bug.

Reply:
I guess I could just hard code it to ignore those specific files. Not ideal, but gets us through this release version at least.